README.md

```diff
@@ -192,6 +192,40 @@ function getCustomClientData() {
 }
 ```
 
+### Building
+
+#### Installation
+Before you can build this you will need to make sure that you are using a node version that matches our target version of node. To see this version you can check the [Jenkinsfile](Jenkinsfile)
+
+##### On Mac
+It would be recommended to manage node version using something like nvm
+```
+brew install nvm
+nvm use 10.15.3
+npm ci
+```
+
+##### On Windows or Linux
+Make sure the correct version of node is being used and run install
```

Comment:
What's the 'correct' version of node?  Was 12.4 just too old?

Reply:
Oh, nevermind, I can see that above... Why is 10.15.3 necessary?

Reply:
10.15.3. 12.4 was too new. I think this had something to do with our dependencies something like a compile of node-gyp failed. I imagine we will eventually want to look into that as well

Reply:
I would just note that consumers should build with the latest node LTS version and leave it at that. We should be making sure it works consistently as new releases come out, and we won't need to update the docs every time there's a new release.

I'd also leave out specific brew/nvm steps for setting up a node version. We can tell folks what version they should use, but there's no real reason for us to maintain instruction on how to manage node.